### PR TITLE
Cran comments values

### PR DESCRIPTION
**Pull request by natesheehan**
I believe I have tended to the resubmission requests - let me know if there is anything else we should/could add to the docs

```
Please add \value to .Rd files regarding exported methods and explain
the functions results in the documentation. Please write about the
structure of the output (class) and also what the output means. (If a
function does not return a value, please document that too, e.g.
\value{No return value, called for side effects} or similar)
Missing Rd-tags:
      ab_save.Rd: \value
      ab_scenario.Rd: \value

Some code lines in examples are commented out in ab_json.Rd
Please never do that. Ideally find toy examples that can be regularly
executed and checked. Lengthy examples (> 5 sec), can be wrapped in
\donttest.
```

### CODE REVIEW

**Comment by Robinlovelace** [comment]
> `Please never do that`

woops :smile:

File: R/ab_scenario.R [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #' in 'long' form, zones and, optionally, buildings from where trips can
 #' start and end as inputs.
 #'
-#' It returns an `sf` object by default representing individual trips
+#' @return An `sf` object by default representing individual trips
```
Comment (Robinlovelace) [inline R/ab_scenario.R:7]: :+1:

File: R/ab_scenario.R [hunk 2]
```diff
@@ -223,7 +220,7 @@ ab_json = function(
 
 #' Save OD data as JSON files for import into A/B Street
 #'
```
Comment (Robinlovelace) [inline R/ab_scenario.R:222]: :+1: the info was in there but just needed the right structure and Roxygen2 tags. Efficient fixes!

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
This looks good and is a clear step in the right direction. I suspect we need to find a commented section in one of the `@examples` though...

**Comment by Robinlovelace** [comment]
I see:

```
rg "# " man                                                  
man/ab_time_normal.Rd
26:# Back to a formal time class

man/ab_scenario.Rd
70:colSums(od[3:7]) # 0.17 vs 0.05 for ab_scenario

man/ab_json.Rd
51:# Legacy code from ActDev project commented out
52:# ab_list$people$trips[[1]]
53:# dutch = ab_scenario(
54:#   leeds_houses,
55:#   leeds_buildings,
56:#   leeds_desire_lines,
57:#   leeds_zones,
58:#   scenario = "godutch",
59:#   output = "sf"
60:# )
61:# ab_list = ab_json(dutch, mode_column = "mode_godutch")
62:# ab_list$scenario
63:# str(ab_list$people$trips[[9]])
64:# # add times
65:# dutch$departure = ab_time_normal(hr = 1, sd = 0, n = nrow(dutch))
66:# ab_list_times = ab_json(dutch)
67:# str(ab_list_times$people$trips[[9]])
68:# ab_save(ab_list_times, f)
69:# readLines(f)[1:30]
70:# 60^2

```

File: R/ab_scenario.R (hunk 2 above)
Comment (natesheehan) [reply R/ab_scenario.R:222]: should we use this then for the `@return`` tag?

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**